packages/grid/_modules_/grid/models/params/gridCellParams.ts

```diff
@@ -28,11 +30,11 @@ export interface GridCellParams {
   /**
    * The column of the row that the current cell belongs to.
    */
-  colDef: any;
+  colDef: GridColDef;
   /**
    * GridApi that let you manipulate the grid.
    */
-  api: any;
+  api: GridApi;
```

Comment:
we could add `undefined` as for datagrid it won't be there

Reply:
Do we have a plan to export different interfaces for `XGrid` and `DataGrid` ?
I can set `api: GridApi | undefined` and let the user of `XGrid` do `api!.getState()` for instance
But if we have a way to generate a different interface it would be nice

Reply:
I agree that it would be nice to generate a different interface but I think at this stage it would add more complexity so I suggest we keep the idea for the next iterations.
I'm working on the split bw the 2 components in #2176, and as mentioned in https://github.com/mui-org/material-ui-x/issues/2189#issuecomment-885546633, I think we should approach the setting of params in a generic way

Reply:
OK to let `api: GridApi` and do another PR with https://github.com/mui-org/material-ui-x/issues/2189#issuecomment-885546633 and `api: GridApi | undefined`

Or do you want `api: GridApi | undefied` even though in reality it is currently never undefined ?

Reply:
I think that one is no longer relevant since it will be removed in https://github.com/mui-org/material-ui-x/issues/1821?